typings/index.d.ts

```diff
@@ -889,7 +889,7 @@ declare module 'discord.js' {
     public syncedAt: number;
     public syncing: boolean;
     public type: string;
-    public user: User;
+    public user: User | null;
     public delete(reason?: string): Promise<Integration>;
     public edit(data: IntegrationEditData, reason?: string): Promise<Integration>;
     public sync(): Promise<Integration>;
```

Comment:
Intention is there. Will is lesser than. Can The will ap be forked then merge through time machine?